bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -162,8 +162,12 @@ public void statusUpdated(final Thing thing, ThingStatusInfo thingStatus) {
                 Bridge bridge = (Bridge) thing;
                 for (Thing bridgeThing : bridge.getThings()) {
                     if (thingStatus.getStatus() == ThingStatus.ONLINE) {
-                        ThingStatusInfo statusInfo = ThingStatusInfoBuilder.create(ThingStatus.ONLINE).build();
-                        setThingStatus(bridgeThing, statusInfo);
+                    	ThingStatusInfo bridgeThingStatus = bridgeThing.getStatusInfo(); 
+                    	if (bridgeThingStatus.getStatus() == ThingStatus.ONLINE || 
```

Comment:
Why checking for ONLINE? If the thing of a bridge is already online, we don´t have to set it to ONLINE again, or?